src/main/java/core/BmpTestBase.java

```diff
@@ -41,4 +42,8 @@ public void cleanup() {
         saveEnvironment();
     }
 
+
+    public ProxyServer getServer() {
```

Comment:
Just make server field protected - when there is an inheritance it is more logical